packages/imperative/src/imperative/src/plugins/utilities/npm-interface/install.ts

```diff
@@ -134,14 +139,85 @@ export async function install(packageLocation: string, registry: string, install
         const pluginImpConfig = ConfigurationLoader.load(null, packageInfo, requirerFunction);
 
         iConsole.debug(`Checking for global team configuration files to update.`);
-        if (PMFConstants.instance.PLUGIN_USING_CONFIG &&
-            PMFConstants.instance.PLUGIN_CONFIG.layers.filter((layer) => layer.global && layer.exists).length > 0)
+        if (PMFConstants.instance.PLUGIN_USING_CONFIG)
         {
             // Update the Imperative Configuration to add the profiles introduced by the recently installed plugin
             // This might be needed outside of PLUGIN_USING_CONFIG scenarios, but we haven't had issues with other APIs before
-            if (Array.isArray(pluginImpConfig.profiles)) {
+            const globalLayer = PMFConstants.instance.PLUGIN_CONFIG.layers.find((layer) => layer.global && layer.exists);
+            if (globalLayer && Array.isArray(pluginImpConfig.profiles)) {
                 UpdateImpConfig.addProfiles(pluginImpConfig.profiles);
-                ConfigSchema.updateSchema({ layer: "global" });
+                const schemaUri = new URL(globalLayer.properties.$schema, pathToFileURL(globalLayer.path));
```

Comment:
Similar comment to other PRs, I'm always scared to use the `$schema` property directly as it could be a valid URL (without the `file://` protocol).

If possible, could we try to do something similar to the `Config#getSchemaInfo()` method?

Same goes for `uninstall`

Reply:
In 3f20c48, I've introduced the same `file` protocol check from #2001 to ensure we do not try to update a web URL schema 😁